packages/runtime/container-runtime/src/summary/summaryFormats.md

```diff
@@ -0,0 +1,150 @@
+
+### Protocol tree
+
+The contents of the protocol tree are:
+
+-   `attributes blob` - Attributes such as the sequence number the summary was taken at.
+-   `quorum members blob` - The members in the quorum at the time summary was taken.
+-   `quorum proposals blob` - The quorum proposals at the time summary was taken.
+-   `quorum values blob` - The quorum values at the time summary was taken.
+-   Other blobs and trees may be added at this level as needed.
+
+### App tree
+
+This is what the ".app" tree looks like which is generated by the container runtime during summary upload. The same is passed to container runtime during snapshot download:
+![appTree](./images/appTree.png)
+
+-   `Container`: The root represents the container or container runtime node. Its contents are described below:
+
+    -   `.metadata blob` - This contains container level metadata such as creation time, create version, etc.
+    -   `.aliases blob` - This contains the list of aliases for data stores and the corresponding internal id.
+    -   `.idCompressor blob` - This contains the data for the ID compressor.
+    -   `.blobs tree` - This contains the snapshot for the attachment blobs added by the blob manager. The individual attachment blobs are present as summary attachments (ISummaryAttachment).
+    -   `gc tree` - This contains the snapshot for garbage collection added by the garbage collector.
+    -   `.channels tree` - This contains the subtrees for all the data stores.
+    -   There may be other blobs and trees added at this level as needed.
+
+-   `Data store`: Data store snapshot trees are under ".channels" subtree of container runtime's tree. Its contents are described below:
+
+    -   `.component blob` - This contains the attributes for a data store such as the package name, snapshot version, etc.
+    -   `.channels tree` - This contains the subtrees for all its DDSs.
+    -   There may be other blobs and trees added at this level as needed.
+
+-   `DDS`: DDS snapshot trees are under ".channels" subtree of its parent data store's tree. Its contents are described below:
+    -   `.attributes blob` - This contains the attributes for a DDS such as the type, snapshot version, etc.
+    -   `.header blob` - This is a blob added by some DDSs and contains its data. Note that all DDSs may not add this.
+    -   A DDS may add other blobs and / or trees to represent its data. Basically, a DDS can write its data in any form
+
+### Summary tree distinction
+
+In the visualization above, a summary tree differs from a snapshot tree in the following way:
+In a summary tree, any node can be a summary handle (`ISummaryHandle`). The following diagram shows this with an example where certain parts of the summary tree are unchanged from the previous summary, and they use a summary handle. It is a zoomed in version of the same app tree as above where nodes where summary handles are marked in red:
```

Comment:
... unchanged from the previous summary and thus use a summary handle

Reply:
Rephrased this section.